pkg/client/tx/client_test.go

```diff
@@ -94,9 +97,18 @@ func TestTxClient_SignAndBroadcast_Succeeds(t *testing.T) {
 		blockClientMock,
 	)
 
+	// Define gas settings for the transaction.
+	gasPrices := cosmostypes.NewDecCoins(cosmostypes.NewDecCoin("upokt", math.NewInt(1)))
+	gasSetting, err := flags.ParseGasSetting("auto")
+	require.NoError(t, err)
```

Comment:
This is duplicated, right - I'm not missing some subtle detail? :sweat_smile: 

Wdyt about factoring this out to an unexported static testutil function that returns a `[]TxClientOption` slice that can be passed/destructured directly to the constructor?